package.json

```diff
@@ -30,6 +30,7 @@
     "@types/jest": "^23.3.13",
     "@types/react": "^16.7.22",
     "gh-pages": "^2.0.1",
+    "prettier": "^2.2.1",
```

Comment:
Is prettier needed if you're using `npx prettier` in the action?

Reply:
no but I believe it makes it available for the local dev.

Reply:
I can check on that tomorrow.